contributing/standards/markdown.md

```diff
@@ -0,0 +1,14 @@
+# Markdown Specification
+
+The following is the way that all Markdown files within Exercism should be structured.
+
+Some of the rules in this document are not yet implemented across Exercism. 
+We welcome PRs to fix them.
+All rules are being added to our CI and linting tools.
+
+## Headings
+- All files must start start with a level-1 heading (`# Some heading text`). 
+- Level-1 headings exist purely for consuming on GitHub or equivelent.
+- If the file is rendered on by the Exercism (e.g. displayed on the website, rendered via the CLI), this heading will be removed, and a contextual heading will be inserted.
+- No heading may decend a level greater than one below the previous (e.g. `##` may only be followed by `###`, not `####`)
+- Beyond the single level-1 (`#`) heading, only level-2 (`##`), level-3 (`###`) and level-4 (`####`) headings may be used.
```

Comment:
Why not level 5 headings?

Reply:
Just because I'm not certain that `#` won't be reduced to `###` somewhere, which would mean `h5` would be reduced to `h7`, which isn't valid HTML. As we have like 5 instances of `h5` anywhere, I think we can do without it.

Reply:
> All files must start start with a level-1 heading

Does this mean we'll have a `# Introduction` header for `introduction.md` documents? If so, I feel bulk PRs incoming :D

Reply:
Yes, yes it does! :)